modules/images/webp-uploads/load.php

```diff
@@ -529,17 +529,18 @@ function webp_uploads_img_tag_update_mime_type( $image, $context, $attachment_id
 
 	$basename = wp_basename( $metadata['file'] );
 	foreach ( $urls as $url ) {
-		if ( isset( $metadata['file'] ) && strpos( $url, $basename ) !== false ) {
-			// TODO: we don't have a replacement for full image yet, issue. See: https://github.com/WordPress/performance/issues/174.
-			continue;
+		$src_filename = wp_basename( $url );
+
+		// Replace the full size image if present.
+		if ( isset( $metadata['file'], $metadata['sources'][ $target_mime ]['file'] ) && strpos( $url, $basename ) !== false ) {
```

Comment:
Based on the above comment:
```suggestion
		if ( isset( $basename, $metadata['sources'][ $target_mime ]['file'] ) && strpos( $url, $basename ) !== false ) {
```

Reply:
Same as above as well.

Reply:
Per your comment in https://github.com/WordPress/performance/pull/195#discussion_r822179813 this is already checked before though, so the check here should be removed entirely as it's redundant.
```suggestion
		if ( isset( $metadata['sources'][ $target_mime ]['file'] ) && strpos( $url, $basename ) !== false ) {
```